### DISCUSSION TITLE
How to verify if migration is complete

### DISCUSSION THREAD

**Discussion by srajappa** [open-metadata/OpenMetadata#18650] [Q&A] [answered]
I have a requirement to check if **migration** is completed so that I can run some downstream events. 

Is there a way to confirm over an API or entry in DB to check migration was completed and successful ?

**Comment by Prajwal214** [comment, marked as answer]
To confirm if a migration is completed and successful, you can check the DATABASE_CHANGE_LOG table in the OpenMetadata database. This table tracks all migrations applied to the target database, including their status.
